tests/components/google_generative_ai_conversation/__init__.py

```diff
@@ -1 +1,31 @@
 """Tests for the Google Generative AI Conversation integration."""
+
+from unittest.mock import Mock
+
+from google.genai.errors import ClientError
+import requests
+
+CLIENT_ERROR_500 = ClientError(
+    500,
+    Mock(
+        __class__=requests.Response,
+        json=Mock(
+            return_value={
+                "message": "Internal Server Error",
+                "status": "internal-error",
+            }
+        ),
+    ),
+)
+CLIENT_ERROR_401 = ClientError(
```

Comment:
Per my comment at https://github.com/home-assistant/core/pull/138246#discussion_r1950279203 this seems to be 400

Reply:
Done